README.md

```diff
@@ -63,6 +97,14 @@ and it should print out the version number, for example:
 1.0.0
 ```
 
+#### Authentication
+
+To test the web / authentication
+
+```
+python -c "import tidy3d.web"
```

Comment:
Is this command supposed to test authentication? I still get the same response 
```
[11:25:02] INFO     Using client version: 1.9.0  
```
with and without a valid authentication file

Reply:
I think this readme is just out of date, sorry. I usually do something like
```py
python -c "import tidy3d.web as web; web.get_info(0)
```
just to test whether the authentication completed successfully.